### ISSUE TITLE
this.pin === undefined

### ISSUE THREAD

**Issue by fivdi** [bryan-m-hughes/raspi-peripheral#1]
If I'm not mistaken, `this.pin` is undefined [here](https://github.com/bryan-m-hughes/raspi-peripheral/blob/1.0.7/index.js#L49).

**Comment by bryan-m-hughes** [comment]
Doh, yes. Originally each peripheral had a single instance, looks like I forgot to refactor that method. This is why I initially wanted to do unit tests before I2C :).

I think it should be something like:
```
for(var i = 0; i < this[pins].length; i++) {
  delete this[pins][i];
}
```

Unfortunately I'm about to leave to go out of town, so I don't have time to implement, test, publish, etc until Monday, but I'll make sure to get right on it then.

**Comment by fivdi** [comment]
I didn't get around to submitting a PR for this at the weekend. It's not a blocker for the I2C code.

**Comment by bryan-m-hughes** [comment]
No worries, I'll try and get to it tomorrow

**Comment by bryan-m-hughes** [comment]
Fixed with v1.1.1 of raspi-peripheral
